sites/feedback/reports/acr-2025-06.json

```
@@ -124,28 +267,60 @@
       "earl:test": [ "WCAG21:status-messages" ],
       "acr:severity": "acr:notRelevant",
       "acr:relevancy": "wbv:acr#comments"
+    },
+    {
+      "@type": "wf:Task",
+      "dct:title": "Improve WET-BOEW Postback plugin",
+      "wf:goalDescription": "The WET-BOEW Postback plugin should be improved so that the confirmation and error messages are announced by all popular screen-readers. The current solution only displays the message which is not announced by JAWS or NVDA because no text change happened.",
+      "dct:created": "2025-06-11",
+      "acr:requirement": "WCAG21:status-messages",
+      "earl:test": [ "WCAG21:status-messages" ],
+      "acr:severity": "acr:moderate",
+      "acr:relevancy": "wbv:acr#usability"
+    },
+    {
+      "@type": "wf:Task",
+      "dct:title": "Add focus to text box on click of No button",
+      "wf:goalDescription": "When users click the No button, give the focus to the newly displayed text box so that users don't have to manually move to it.",
+      "dct:created": "2025-06-11",
+      "acr:requirement": "WCAG21:focus-order",
+      "earl:test": [ "WCAG21:focus-order" ],
+      "acr:severity": "acr:moderate",
+      "acr:relevancy": "wbv:acr#usability"
+    },
+    {
+      "@type": "wf:Task",
+      "dct:title": "Enhance redacted text screen-reader alternative",
+      "wf:goalDescription": "Enhance the screen-reader text of the redacted text so that it reads \"personal information redacted\" instead of just \"redacted\".",
+      "dct:created": "2025-06-11",
+      "acr:requirement": "WCAG21:non-text-content",
+      "earl:test": [ "WCAG21:non-text-content" ],
+      "acr:severity": "acr:trivial",
+      "acr:relevancy": "wbv:acr#usability"
     }
   ],
 
   "dct:source": "act:standard/sets/wcag2x/wcag21_all_levelAA.json",
   "acr:conformity": [
     {
       "acr:requirement": "WCAG21:non-text-content",
-      "acr:conformance": "acr:satisfied",
+      "acr:conformance": "acr:furtherTestNeeded",
       "acr:assessment": [
-        "https://wet-boew.github.io/GCWeb/sites/feedback/reports/pre-a11y-2.json"
+        "https://wet-boew.github.io/GCWeb/sites/feedback/reports/pre-a11y-2.json",
+        "https://wet-boew.github.io/GCWeb/sites/feedback/reports/a11y-4.json"
       ],
       "earl:subject": "_:subject",
-      "dct:description": "",
-      "dct:modified": "2023-12-15",
+      "dct:description": "Further testing will be needed to determine whether the non-text content for redacted text is suffucient.",
```

Add a reference to the working task.

```suggestion
      "dct:description": "Further testing will be needed to determine whether the non-text content for redacted text is suffucient. See working task 'Enhance redacted text screen-reader alternative'",
```